src/Widgets/Tooltip.vala

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2021 elementary, Inc (https://elementary.io)
+ *           2021 José Expósito <jose.exposito89@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * Clutter actor to display text in a tooltip-like component.
+ *
+ * Style constants from:
+ * https://github.com/elementary/stylesheet/blob/master/src/widgets/_tooltips.scss
+ */
+public class Gala.Tooltip : Clutter.Actor {
+    private const double BACKGROUND_COLOR = (26 / 255); // #1a
+    private const double BACKGROUND_OPACITY = 0.9;
+    private const double BACKGROUND_BORDER_RADIUS = 3;
+    private const string COLOR = "#ffffff";
```

Comment:
This definitely is not needed. We should not store a string just to parse it everytime we redraw the tooltip. You can statically get an instance to a white color via `Clutter.Color.get_static (Clutter.StaticColor.WHITE)`.

Reply:
I prefer to keep the string because, as this is copy and pasted from the tooltip stylesheet CSS, it makes it easier to change it in the future if a different kind of white is used. Moved to the `static construct` though:
https://github.com/elementary/gala/pull/1076/files#diff-ed4232dda49f45599a7a209514eb953072ee8cc2515630daa9bef8719be1607cR63